todo/src/repo/repo.ts

```diff
@@ -0,0 +1,109 @@
+import { Task } from "../todo";
+
+interface reposotoryPattern<T> {
+    find(findTodo: String),
+    findAll(),
+    create(makeATodo: T), 
+    del(delTask: String),
+    updateTodo(updateTask: String, payload: T),
+    incompleteTodos(),
+    completeTodos(),
+    orderDates(),
+}
+
+class TodoRepo implements reposotoryPattern<Task> {
+
+    private task: Task[] = [];
+
+    // GET all tasks
+    public findAll() {
+        return this.task
+    }
+
+    // POST
+    public create(newTask) {
+        this.task.push(newTask);
+        return newTask
+    }
+
+    // GET a single task
+    public find(uuid) {
+        const found = this.task.filter(tasks => tasks.uuid == uuid)
+        if(found) {
+            return(found);
+        } else {
+            return (`No task with the ID of ${uuid}`)
+        } 
+    }
```

Comment:
If these functions were given a strong return type you would have an error as you are returning both the task and a string conditionally